Beam/UI/Generic UI/Base Classes/DynamicDisplayModeView.swift

```diff
@@ -32,7 +32,7 @@ protocol DynamicDisplayModeView: class {
 extension DynamicDisplayModeView {
     
     func registerForDisplayModeChangeNotifications() {
-        NotificationCenter.default.addObserver(self, selector: Selector("displayModeDidChangeNotification:"), name: .DisplayModeDidChange, object: nil)
+        NotificationCenter.default.addObserver(self, selector: Selector(("displayModeDidChangeNotification:")), name: .DisplayModeDidChange, object: nil)
```

Comment:
Why the double brackets?

Reply:
To temporarily silence the warning in Xcode 11.2.1